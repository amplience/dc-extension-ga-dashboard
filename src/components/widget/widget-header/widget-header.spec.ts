import { render } from '@testing-library/svelte';
import WidgetHeader from './widget-header.svelte';

describe('WidgetHeader', () => {
  it('should render the Widget component', () => {
    const { container } = render(WidgetHeader, {
      title: 'widget-header-title',
    });

    expect(container.firstChild).toMatchSnapshot();
  });
});
