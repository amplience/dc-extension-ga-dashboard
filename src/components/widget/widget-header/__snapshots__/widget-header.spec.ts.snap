// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetHeader should render the Widget component 1`] = `
<div>
  <section
    class="svelte-1kld2ef"
  >
    <div
      class="title svelte-1kld2ef"
    >
      <h2
        class="svelte-1kld2ef"
      >
        widget-header-title
      </h2>
       
    </div>
     
    <div
      class="actions svelte-1kld2ef"
    />
  </section>
</div>
`;
